gulpfile.js

```diff
@@ -2,7 +2,8 @@
 var gulp = require('gulp'),
     jshint = require('gulp-jshint'),
     concat = require('gulp-concat'),
-    Jasmine = require('jasmine');
+    Jasmine = require('jasmine'),
+    Promise = require('bluebird');
```

Comment:
It would be nice to import it only if platform not supporting native `Promise`:
```js
var Promise = typeof Promise === 'undefined' ? require('promise') : Promise,
```